### DISCUSSION TITLE
Load all tailwind classes

### DISCUSSION THREAD

**Discussion by fcrozatier** [tailwindlabs/tailwindcss#14443] [Help]
For an HTML+CSS playground working with tailwind, I need a way to load all tailwind classes since dynamic class names don't work.

Is there a way to ensure all classes are loaded?

**Comment by wongjn** [comment]
This would not really be possible for v3 – there's an infinite amount of classes possible (due to the arbitrary features) and even without these, if you really wanted every single class possible, you would need to generate every possible combination of the variants, (like `dark:first:hover:focus:md:visited:last:only:…:m-1`) so there would be all the "base" classes plus all these permutations of variants and thus the total number of possible classes would be something like (70![^1] + 1) × number of "base" classes (maths probably not correct here, but something like this). The "real" number would be slightly less since some variants don't make sense together, i.e. `sm:md:lg:xl:m-1` but this obscenely large number should demonstrate the point.

[^1]: 70 factorial, equal to 11978571669969891796072783721689098736458938142546425857555362864628009582789845319680000000000000000

**Reply by fcrozatier** [reply]
Yeah this makes sense, the naive approach would not work. 
To achieve what I want I'll probably have to load a tailwind wasm engine, this seems to be how it's done here https://play.tailwindcss.com/
